### PR TITLE
implemented readStream() interface

### PR DESCRIPTION
**Pull request by isaackwan**
Hello,

I've implemented the readStream() interface. Sorry for not following the standard PR template -- will update the PR once this is ready. Meanwhile, please take a look at my code and see if it appears OK to you.

BTW -- can you shine some light on the tests? From what I read from the code a good chunk of them are just testing if the function calls returns `false`...?

### CODE REVIEW

File: src/BackblazeAdapter.php [hunk 1]
```diff
@@ -101,7 +102,19 @@ public function read($path)
      */
     public function readStream($path)
     {
-        return false;
```
Comment (mhetreramesh) [inline src/BackblazeAdapter.php:104]: This looks good to me but I can't see tests for it. I'll prefer to write test while writing any function.

**Comment by mhetreramesh** [comment]
@isaackwan Thanks for your contribution. I'm writing tests whenever I  get a chance to write logic in the actual function. This adapter is getting evolved month over month. Eventually, we will have a complete package. 

For now, I'm going to approve this PR. Would recommend you to write tests if possible for further enhancement.

Thanks

**Comment by mhetreramesh** [comment]
Also, will recommend starting using travis-ci & scrutinizer for your forked repo's so you will come to if anything is broken before you raise a PR.

**Comment by isaackwan** [comment]
I've just given it a few thoughts, and I personally think that it would
actually be better to declare a hard dependency on `guzzlehttp/psr7`
instead of requiring on the PSR-7 interface... Do you think that it would
be better for me to start a new PR, make a new commit or start a new
squashed commit?

**Comment by mhetreramesh** [comment]
I like the idea of using PSR-7. Will read about it more. Thanks for the suggestion!

**Pull request merged**